.cirrus.yml

```diff
@@ -92,3 +90,30 @@ task:
 
   test_script:
     - pytest -v
+
+codecov_task:
+  auto_cancellation: true
+
+  depends_on:
+    - linux
+    - osx
+    - windows
+
+  container:
+    image: python:3.6
+
+  env:
+    CODECOV_TOKEN: ENCRYPTED[be41b51dac12576cfe9889ddd604d1f9db97cee5db0d98f7241fc8d5adf8d23da1135cd67e20b2ecec4005ac03083286]
+
+  system_script:
+    - apt-get update
+    - apt-get install -y libgl1-mesa-glx libqt5x11extras5
+
+  install_script:
+    - python -m pip install --retries 3 --upgrade pip
+    - pip install -r requirements.txt
+    - pip install pytest-cov codecov
+
+  coverage_script:
+    - pytest --cov=./napari --collect-only
```

Comment:
@kne42 are you sure about the 'collect-only'?

Reply:
nope, I can try and we can see if that changes anything ;P